packages/scheduler/src/forks/SchedulerNoDOM.js

```diff
@@ -71,7 +71,7 @@ var isHostTimeoutScheduled = false;
 
 let getCurrentTime;
 const hasPerformanceNow =
-  typeof performance === 'object' && typeof performance.now === 'function';
+  window.performance && typeof performance === 'object' && typeof performance.now === 'function';
```

Comment:
same question as above here.   Where does performance come from?

Reply:
Yeah, I find the reason. In my case, performance was set 'null' some where, so 
    typeof` performance === 'object'  
check pass, but 
    typeof performance.now === 'function' 
cause an error